ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -274,6 +289,10 @@ export default class StatusDropdown extends React.Component {
                     Mark as fixed
                   </DropdownItem>
                 )}
+
+                <DropdownItem onClick={() => this.toggle('showTagsModal')}>
+                  {!alertSummaryActiveTags.length ? 'Add tags' : 'Edit tags'}
```

Comment:
I'm curious what tags are - it seems a bit vague. Are they labels? Would a tooltip with a more verbose explanation make it clearer?